src/utils/countriesData.json

```diff
@@ -0,0 +1,1752 @@
+[
+  {
+    "countryName": "New Zealand",
+    "countryCode": "NZ",
+    "currencyName": "New Zealand Dollars",
+    "currencyCode": "NZD",
+    "currencyCountryFlag": "NZ"
+  },
```

Comment:
If you ever want to use translations for the country names, you need this file in every language we want to support. This is still an unsolved problem in the Treecounter app yet (but there are free sources on the internet providing this information).

Reply:
We will add this later